.github/workflows/release.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
           python-version: ${{ matrix.python-versions }}
       - uses: addnab/docker-run-action@v3
         with:
-            image: valory/open-autonomy-user:latest
+            image: valory/open-autonomy-user:0.10.0.post2
```

Comment:
The `latest` tag was causing issues locally when trying to test the workflow as an older version of the image was being used.

Better to explicitly tag and update every time we bump.